themes/mongodb/static/navbar.js

```diff
@@ -398,6 +461,7 @@ $(function() {
     setupFastLoad();
     updateVersionSelector();
     createCopyButtons();
+    setupTabs();
```

Comment:
Also need this on line 353.

(I'm thinking about putting in place a better architecture so this duplication isn't necessary, but for now...)